rest_api/operator_apis/ingresscontroller-operator-openshift-io-v1.adoc

```diff
@@ -1607,10 +1624,10 @@ Type::
 
 | `clientAccess`
 | `string`
-| clientAccess describes how client access is restricted for internal load balancers. 
- Valid values are: * "Global": Specifying an internal load balancer with Global client access   allows clients from any region within the VPC to communicate with the load   balancer. 
-     https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing#global_access 
- * "Local": Specifying an internal load balancer with Local client access   means only clients within the same region (and VPC) as the GCP load balancer   can communicate with the load balancer. Note that this is the default behavior. 
+| clientAccess describes how client access is restricted for internal load balancers.
+ Valid values are: * "Global": Specifying an internal load balancer with Global client access   allows clients from any region within the VPC to communicate with the load   balancer.
+     https://cloud.google.com/kubernetes-engine/docs/how-to/internal-load-balancing#global_access
+ * "Local": Specifying an internal load balancer with Local client access   means only clients within the same region (and VPC) as the GCP load balancer   can communicate with the load balancer. Note that this is the default behavior.
```

Comment:
- Use blank lines.
- Should VPC be spelled out?
- Tick the values, removing the quotation marks.